src/renderer/Instance.ts

```diff
@@ -114,16 +114,15 @@
 			return proc;
 		}
 	}
-	public install(): Promise<TaskRuntime<Task.State>> {
-		return new Promise((resolve, reject) => {
+	public async install(): Promise<TaskRuntime<Task.State>> {
+		return await (new Promise((resolve, reject) => {
 			this.isInstalling = true;
 			const location: MinecraftLocation = MinecraftFolder.from(path.join(app.getPath("userData"), "./game/"));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This assertion is unnecessary since it does not change the type of the expression.](https://app.codacy.com/manual/lukechu10/Minecraft-Box-Launcher/pullRequest?prid=5396733)